pkg/fluentd/router/pluginstore.go

```diff
@@ -30,7 +30,7 @@ type LabelRouter struct {
 	Routes []*Route `json:"routes,omitempty"`
 }
 
-// Each fluentd config instance will create a route pluginstore.
+// NewRoutePlugin Each fluentd config instance will create a route pluginstore.
```

Comment:
```suggestion
// Each fluentd config instance will create a route pluginstore.
```

Reply:
Add variable descriptions before comments, which is a standard format requirement

Reply:
No need to add function name like NewRoutePlugin and NewGlobalRouter to the function's comments

Reply:
Using the function or parameter name as the beginning of the comment is correct, but we need to ensure that the comment is grammatical. .

Reply:
Seems that both comments are not grammatical:
```
NewRoutePlugin Each fluentd config instance will create a route pluginstore.
NewGlobalRouter The global router to store routes Member
```

Reply:
<img width="764" alt="image" src="https://user-images.githubusercontent.com/18587688/218986873-93c533ad-9983-477a-a13f-e95b198cda9c.png">

Reply:
> NewRoutePlugin Each fluentd config instance will create a route pluginstore.
> NewGlobalRouter The global router to store routes Member

I mean we cannot simply append a function name to the original comments to conform with the guideline, we need to make the entire comment conform the grammar maybe like this:

```shell
NewRoutePlugin will create a route pluginstore for each fluentd config instance.
NewGlobalRouter will create a global router to store routes Member
```

Reply:
updated